go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/docker/swarmkit
+
+go 1.12
+
+require (
+	code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Microsoft/go-winio v0.4.14
+	github.com/beorn7/perks v0.0.0-20190414131216-e7f67b54abbe // indirect
+	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
+	github.com/coreos/etcd v3.3.12+incompatible
+	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7 // indirect
```

Comment:
This package tags releases, but they're not SemVer (https://github.com/coreos/go-systemd/releases); is there a way to show that this matches a tag and/or force go mod to use a tag?

Reply:
No :( see previous comment. We can manually tell it to use a tag when running `go get`, but that's about it.